ap_src/ap_app/utils/switch_permissions.py

```diff
@@ -0,0 +1,69 @@
+# Models
+from ..models import (UserProfile)
+from django.contrib.auth.models import User
+
+# Utils
+from teams_utils import get_users_sharing_teams
+from user_profile import get_user_id_from_username
+from user_profile import get_userprofile_id_from_user_id
+
+# this check should be activated when the user leaves a team
+def check_for_lingering_switch_perms(username): # stops users from having switch perms when they don't share any teams
+    user_id = get_user_id_from_username(username)
+    print("user_id:", user_id)
```

Comment:
This print statement was for debugging. It is no longer needed so it will be removed.

Fixed with 56e2c3b

Reply:
This print statement was to prove the algorithm implemented actually works as expected. It is no longer needed so it will be commented out. However, the `if` statement and general logic here is useful (and necessary) as the code for removing redundant permissions actually needs to be implemented at some point. This would fix the issue being worked on.

Fixed with 56e2c3b